src/julia_atomics.h

```diff
@@ -56,6 +56,15 @@ enum jl_memory_order {
     jl_memory_order_seq_cst
 };
 
+/**
+ * Cache line size
+*/
+#if defined(_CPU_AARCH64_) && defined(_OS_DARWIN_) // Apple silicon has 128 cache lines
```

Comment:
So this would be wrong for a Linux system running on Apple Silicon (e.g. Asahi Linux)?

Reply:
yes.